### PR TITLE
en: add tidbmonitor remote write

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added, or deleted? (Required)

Add tidbmonitor remote write

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1399
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/aggregate-multiple-cluster-monitor-data.md [hunk 1]
```diff
@@ -53,9 +51,9 @@ Thanos provides [Thanos Query](https://thanos.io/tip/components/query.md/) compo
 
        In the command above, `${thanos_namespace}` is the namespace where the Thanos Query component is deployed.
 
-In Thanos Query, a Prometheus instance corresponds to a store and also corresponds to a TidbMonitor. After deploying Thanos Query, you can provide a uniform query interface for monitoring data through the Thanos Query's API.
+In Thanos Query, a Prometheus instance corresponds to a store and also corresponds to a TidbMonitor. After deploying Thanos Query, you can provide a uniform query interface for monitoring data through Thanos Query's API.
 
-## Access the Thanos Query Panel
+### Access the Thanos Query Panel
```
Comment (qiancai) [inline en/aggregate-multiple-cluster-monitor-data.md:56]:
```suggestion
### Access the Thanos Query panel
```

File: en/aggregate-multiple-cluster-monitor-data.md [hunk 2]
```diff
@@ -145,3 +143,38 @@ stringData:
         enable: true
       part_size: 41943040
 ```
+
+## RemoteWrite mode
+
+Apart from aggregating data via Thanos Query, you can also push monitoring data to Thanos using Prometheus' RemoteWrite feature.
```
Comment (qiancai) [inline en/aggregate-multiple-cluster-monitor-data.md:149]:
```suggestion
Besides aggregating data via Thanos Query, you can also push monitoring data to Thanos using Prometheus' RemoteWrite feature.
```

Comment (qiancai) [reply en/aggregate-multiple-cluster-monitor-data.md:149]: Apart from can mean either 'including' or 'excluding' ，需要去判断

**Review by qiancai — COMMENTED** [review]
rest LGTM

File: en/aggregate-multiple-cluster-monitor-data.md (hunk 2 above)
Comment (ran-huang) [reply en/aggregate-multiple-cluster-monitor-data.md:149]: Got it. Thanks!

**Review by qiancai — APPROVED** [review]

**Pull request merged**